components/gc-features/gc-features-en.html

```diff
@@ -1,705 +1,206 @@
+					<p>Brief description of the feature being promoted. Brief description of the feature being promoted.</p>
+				</div>
 			</div>
-		</div>
-	</div>
-</section>
-
-<h3>Feature - Markup change - v9.5.0</h3>
-<section class="gc-features">
-	<h2>One tile - Promotional feature</h2>
-	<div class="row">
-		<div class="col-md-8">
-			<div class="row">
-				<div class="col-sm-6">
-					<img src="img/feature-360x203.png" alt="" class="thumbnail">
+			<div class="col-lg-4 col-sm-6">
+				<div class="well well-sm eqht-trgt">
+					<img src="img/feature-360x203.png" alt="">
+					<h3><a href="#" class="stretched-link">[Feature hyperlink text]</a></h3>
+					<p>Brief description of the feature being promoted.</p>
 				</div>
-				<div class="col-sm-6">
-					<h3><a class="stretched-link" href="#">[Feature hyperlink text]</a></h3>
+			</div>
+			<div class="col-lg-4 col-sm-6">
+				<div class="well well-sm eqht-trgt">
+					<img src="img/feature-360x203.png" alt="">
+					<h3><a href="#" class="stretched-link">[Feature hyperlink text]</a></h3>
 					<p>Brief description of the feature being promoted.</p>
 				</div>
 			</div>
 		</div>
-	</div>
-</section>
+	</section>
 
-<h2>Previous design patterns</h2>
-<p>Actual visual rendering of previous design patterns may vary due to recent changes.</p>
-<details>
-	<summary>Context-specific features - Removal - 6.1</summary>
+	<h3>One-tile feature</h3>
 	<section class="gc-features">
-		<h2>Features</h2>
-		<div class="row wb-eqht">
-			<div class="col-lg-4 col-md-6 mrgn-bttm-md">
-				<h3 class="h5"><a href="#" class="stretched-link">[Feature hyperlink text]</a></h3>
-				<img src="img/feature-360x203.png" alt="" class="img-responsive thumbnail mrgn-bttm-sm">
-				<p>Brief description of the feature being promoted.</p>
-			</div>
-			<div class="col-lg-4 col-md-6 mrgn-bttm-md">
-				<h3 class="h5"><a href="#" class="stretched-link">[Feature hyperlink text]</a></h3>
-				<img src="img/feature-360x203.png" alt="" class="img-responsive thumbnail mrgn-bttm-sm">
-				<p>Brief description of the feature being promoted.</p>
-			</div>
-			<div class="col-lg-4 col-md-6 mrgn-bttm-md">
-				<h3 class="h5"><a href="#" class="stretched-link">[Feature hyperlink text]</a></h3>
-				<img src="img/feature-360x203.png" alt="" class="img-responsive thumbnail mrgn-bttm-sm">
-				<p>Brief description of the feature being promoted.</p>
+		<h2>Featured</h2>
+		<div class="row">
+			<div class="col-md-8">
+				<div class="row">
+					<div class="col-sm-6">
+						<img src="img/feature-360x203.png" alt="" class="thumbnail">
+					</div>
+					<div class="col-sm-6">
+						<h3><a class="stretched-link" href="#">[Feature hyperlink text]</a></h3>
+						<p>Brief description of the feature being promoted.</p>
+					</div>
+				</div>
 			</div>
 		</div>
 	</section>
-</details>
+</section>
 
-<details>
-	<summary>GC activities and initiatives - Removal - v6.1</summary>
+<h2>Government initiatives (variant)</h2>
+<section>
```

Comment:
We can not have a double section. A section must contains an heading and here (at this line) implicit section are sufficient.